src/AWS.Deploy.Orchestration/AWS.Deploy.Orchestration.csproj

```diff
@@ -22,6 +22,7 @@
     <PackageReference Include="AWSSDK.CloudFormation" Version="3.7.7.14" />
     <PackageReference Include="AWSSDK.S3" Version="3.7.1.17" />
     <PackageReference Include="AWSSDK.AppRunner" Version="3.7.3.11" />
+    <PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="3.7.16" />
```

Comment:
Could you update our internal license tooling and the THIRD_PARTY_NOTICES file with the new dependency?

Reply:
Done